charts/gardener-dashboard/charts/runtime/templates/bootstrapper/deployment.yaml

```diff
@@ -0,0 +1,123 @@
+{{- if .Values.global.bootstrapper.enabled }}
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: gardener-dashboard-terminal-bootstrapper
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: gardener-dashboard
+    app.kubernetes.io/component: terminal-bootstrapper
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: gardener-dashboard
+      app.kubernetes.io/component: terminal-bootstrapper
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: gardener-dashboard
+        app.kubernetes.io/component: terminal-bootstrapper
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        app.kubernetes.io/instance: "{{ .Release.Name }}"
+        app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+        {{- if .Values.global.bootstrapper.podLabels }}
+        {{- toYaml .Values.global.bootstrapper.podLabels | nindent 8 }}
+        {{- end }}
+      annotations:
+        checksum/configmap-gardener-dashboard-terminal-bootstrapper-config: {{ include (print .Template.BasePath "/bootstrapper/configmap-config.yaml") . | sha256sum }}
+        {{- if .Values.global.bootstrapper.kubeconfig }}
+        checksum/secret-gardener-dashboard-terminal-bootstrapper-kubeconfig: {{ include (print .Template.BasePath "/bootstrapper/secret-kubeconfig.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if .Values.global.bootstrapper.podAnnotations }}
+        {{- toYaml .Values.global.bootstrapper.podAnnotations | nindent 8 }}
+        {{- end }}
```

Comment:
also add this to the dashboard deployment to be consistent and add documentation in `values.yaml`:
`# podAnnotations: # YAML formated annotations used for pod template`